.github/workflows/install-build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+sudo apt update
+sudo apt install -y libboost1.62-all-dev \
+	libglew-dev \
+	libogg-dev \
+	libopenal-dev \
+	libsdl2-dev \
+	libvorbis-dev \
+	cppcheck \
+	doxygen \
+	python3-pip \
+	ccache
+python3 -m pip install flake8==3.7.9
```

Comment:
Is it a part of the Linux build?  We have a separate build for linters, probably we don't need this here.

Reply:
Renamed to `linux-install-build.sh` and removed linter installation.